### ISSUE TITLE
Tutorial on using Modernizr.supports

### ISSUE THREAD

**Issue by stucox** [Modernizr/Modernizr#1226]
https://twitter.com/jpdevries/status/436058155445784576

It certainly isn't obvious how our detect for `@supports` could be used. We should write an article with some use cases, with example code, and could also explain how we use it internally.

**Comment by sterlingwes** [comment]
It also returns false in Safari 8 (no `window.CSS`)

Using `modernizr v3.0.0-alpha.4` custom build

**Comment by patrickkettner** [comment]
thats because it [doesn't exist in safari 8](http://caniuse.com/#search=supports)

**Comment by sterlingwes** [comment]
Yup @patrickkettner , is there a reason it's added by default in the custom build whether I add that test or not (I haven't added it).

**Comment by sterlingwes** [comment]
Further to that:

```
"modernizr": {
    "options": ["mq"],
    "feature-detects": [
      "canvas",
      "css/animations",
      "css/backgroundposition-shorthand",
      "css/backgroundposition-xy",
      "css/backgroundrepeat",
      "css/backgroundsize",
      "css/backgroundsizecover",
      "css/boxshadow",
      "css/flexbox",
      "css/mediaqueries",
      "css/opacity",
      "css/rgba",
      "css/transforms",
      "css/transforms3d",
      "css/transitions",
      "css/vhunit",
      "css/vwunit",
      "svg",
      "requestanimationframe"
    ]
  }
```

**Comment by patrickkettner** [comment]
Sorry, not following what is happening. Could you elaborate?

**Comment by sterlingwes** [comment]
I'm using the [custom build api](https://github.com/Modernizr/Modernizr#building) with the above feature-detects. I assume `CSS Supports` is added by virtue of the CSS feature tests I've specified (as some sort of dependent), but these tests all pass in Safari save for CSS supports (as you mention above). I comment out that test in my custom build, just don't think it's a result that should be expected given my config.

Thanks for your time!

**Comment by patrickkettner** [comment]
ah! That is because the `transforms3d` detect has the `@supports` detect as a requirement. brings up an interesting point on whether or not we should be be hide detect data for ones not explicitly requested, though...
